packages/nextjs/app/builders/0x391b638EB7d21B122Be2Ed69eb81DA7E0F168177/page.tsx

```diff
@@ -0,0 +1,83 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+import { FaGithub, FaTwitter } from "react-icons/fa";
+import { Address } from "~~/components/scaffold-eth";
+
+const WalletAddress = "0x391b638EB7d21B122Be2Ed69eb81DA7E0F168177";
+const socialLinks = [
+  {
+    href: "https://github.com/Ebukamoses",
+    label: "GitHub",
+    icon: <FaGithub />,
+  },
+  {
+    href: "https://twitter.com/bukan_moses",
+    label: "Twitter",
+    icon: <FaTwitter />,
+  },
+];
+
+const EbukaMosesProfile: NextPage = () => {
+  return (
+    <div className="flex flex-col items-center min-h-screen bg-gradient-to-b from-base-200 to-base-300 px-4 sm:px-8 py-4 sm:py-8">
+      <div className="max-w-[400px] w-full bg-gradient-to-r from-gray-[#f5f5f5] via-white to-[#e6e6e6] dark:from-base-100 dark:via-base-200 dark:to-base-100 shadow-md rounded-2xl p-4 sm:p-10 border border-gray-100 dark:border-accent hover:shadow-gray-200/20 transition-all duration-300">
```

Comment:
ou should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.